src/Symfony/Component/Serializer/Normalizer/DenormalizerInterface.php

```diff
@@ -27,6 +27,13 @@
      * @param array  $context options available to the denormalizer
      *
      * @return object
+     *
+     * @throws \Symfony\Component\Serializer\Exception\BadMethodCallException
+     * @throws \Symfony\Component\Serializer\Exception\InvalidArgumentException
+     * @throws \Symfony\Component\Serializer\Exception\UnexpectedValueException
+     * @throws \Symfony\Component\Serializer\Exception\ExtraAttributesException
+     * @throws \Symfony\Component\Serializer\Exception\LogicException
+     * @throws \Symfony\Component\Serializer\Exception\RuntimeException
```

Comment:
@fabpot that's a good point. I'll document the "why" exceptions are used for. Currently, it's not easy to get it at first glance.